capeinfra/meta/capemeta.py

```diff
@@ -32,10 +33,44 @@ def __init__(self, name, **kwargs):
                 source=FileAsset(etl_def["srcpth"]),
             )
 
+        self.capepy = CapePy(self.automation_assets_bucket)
+
         # We also need to register all the expected outputs for this component
         # resource that will get returned by default.
         self.register_outputs(
             {
                 "cape-meta-automation-assets-bucket": self.automation_assets_bucket.bucket
             }
         )
+
+
+class CapePy(CapeComponentResource):
+    def __init__(self, assets_bucket: VersionedBucket, **kwargs):
+        self.name = "capepy"
+        super().__init__(
+            "capeinfra:meta:capemeta:CapePy",
+            self.name,
+            desc_name="Resources for distributing the CapePy library",
+            **kwargs,
+        )
+
+        self.bucket = assets_bucket
+        self.object = self.bucket.add_object(
+            f"{self.name}-object",
+            key="capepy.whl",
+            source=FileAsset("./assets/capepy/capepy-0.1.0-py3-none-any.whl"),
```

Comment:
Maybe/maybe not...should the version of the library be a config value so we can change it without changing code?

Reply:
this whole loading process will need to change entirely anyway. I think we should punt this to another PR where we actually figure out how we want to package and ship this around

Reply:
Totally fine with me if we have an issue to track that

Reply:
Moved to an issue: #179

Also put a potential plan as I was brainstorming about this.